ctlearn/default_models/single_cnn.py

```diff
@@ -8,7 +8,10 @@ def single_cnn_model(data, model_params):
 
     # Load neural network model
     network_input_img = tf.keras.Input(shape=data.img_shape, name=f"images")
```

Comment:
I'm not sure if we want to support MLP only without the CNNs here or in a separate file. There is no need for an additional file I guess. @nietootein What are your thoughts about this?
I ran a quick check. It is possible to request only the parameter list without the images from dl1dh.  @sahilyadav27 So I guess you can just check here if images are selected or not with `data.img_shape != None`

Reply:
I thought having a separate MLP model would make sense because we might want to add the MLP model to other models as well like CNN_RNN and ResNet. So it would be better to have a common MLP model independent of the image model rather than having it inside the Single CNN model?

Reply:
This should work now